src/main/java/duke/storage/ExportSelection.java

```diff
@@ -0,0 +1,125 @@
+package duke.storage;
+
+import com.opencsv.CSVWriter;
+
+import duke.exceptions.DukeException;
+import duke.models.locker.Locker;
+
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ExportSelection {
+    private static final String CSV_OUTPUT_PATH = "export_selected.csv";
```

Comment:
i was thinking of using only one file.  Like just one export.csv

Reply:
fixed